content/blog/2018-03-13-weathercan.md

```diff
@@ -0,0 +1,500 @@
+
+This is one of the reasons why, when I designed [`weathercan`](http://github.com/ropensci/weathercan), I tried as hard as possible to make it simple and straightforward. `weathercan` is an R package designed to make it easy to access historical weather data from [Environment and Climate Change Canada (ECCC)](http://climate.weather.gc.ca/historical_data/search_historic_data_e.html). It downloads, combines, cleans, and transforms the data from multiple stations and across long time frames. So when you access ECCC data, you get everything in one dataset. Nifty, eh?
+
+However, there is a certain point at which the user has to take matters into their own hands. Although downloading data with `weathercan` is fairly straight forward, weather data often needs to be integrated into other data sets. Depending on the other data this can be a tricky step. You may want to combine `weathercan` data with other types of measurements (e.g., river water samples on a specific day), or summarize and join it with data on other scales (e.g. temporal or spatial).
+
+Integrating `weathercan` data can be straightforward, but often it's a bit convoluted. That's why I'm going to walk you through some different ways of integrating weather data from `weathercan` with other data sets.
+
+We'll also be using several other R packages to do this, so why don't we load them right now:
+
+``` r
+# Data manipulation and plotting
+library(dplyr)
+library(ggplot2)
+
+# Checking data completeness
+library(naniar)
+
+# Access to data containing feeder visits by birds
+library(feedr)
+
+# Spatial analyses
+library(sf)
+library(mapview)
+```
+
+Well, I've told you it's easy to get data from `weathercan`, so let's start by doing so. For example, if you wanted to download weather data for all of Manitoba, Canada since the New Year, you have only three steps:
+
+1.  Load the package:
+
+    ``` r
+    library(weathercan)
+    ```
+
+2.  Look at the built in `stations` data set to find the specific stations you're interested in (you can also use the `stations_search()` function). Here, we'll use the [`dplyr`](http://dplyr.tidyverse.org/) package (part of [`tidyverse`](https://www.tidyverse.org)) to `filter()` stations to only those in the province of Manitoba, at daily intervals, and which have an end date of at least 2018 (which likely means it's still operational). Note that we'll also be removing some columns (`prov`, `climate_id`, `WMO_id`, `TC_id`) just for clarity.
```

Comment:
our fault - notice in preview that the #2 in list doesn't align with top text of that list item. Sorry about that, it's on our to-fix list